src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content-adjusted/elements.js

```diff
@@ -0,0 +1,20 @@
+import Card from '@material-ui/core/Card';
+import styled from 'styled-components';
+
+const ChartContainer = styled.div`
```

Comment:
Can we use the ChartContainer in commons here? I'm not able to remember if it's the same, but I think it can be used.

Reply:
Yes, we can. Done!